solr/core/src/test/org/apache/solr/request/TestFaceting.java

```diff
@@ -931,5 +934,28 @@ public void testListedTermCounts() throws Exception {
         "//lst[@name='facet_fields']/lst[@name='title_ws']/int[2][@name='Book2']",
         "//lst[@name='facet_fields']/lst[@name='title_ws']/int[3][@name='Book3']");
   }
+  
+  @Test
+  public void testFacetCountsWithMinExactHits() throws Exception {
+    final int NUM_DOCS = 20;
+    for (int i = 0; i < NUM_DOCS ; i++) {
+      assertU(adoc("id", String.valueOf(i), "title_ws", "Book1"));
+      assertU(commit());
```

Comment:
Actually, I wanted to have multiple segments. I could do something like "sometimes()", but since the number of docs is low, I didn't think it was needed to add any randomization or more complex logic.